source/git-util.js

```diff
@@ -8,6 +8,11 @@ exports.latestTag = async () => {
 	return stdout;
 };
 
+exports.newFilesSinceLastRelease = async () => {
+	const {stdout} = await execa('git', ['diff', '--stat', '--diff-filter=A', await this.latestTag(), 'HEAD']);
```

Comment:
The output of this command should be parsed here, not where this method is used.

Reply:
Done